src/actions/lender.ts

```diff
@@ -4,16 +4,17 @@ import BN from 'bn.js'
 export function LenderActions<ActionBase extends Constructor<TinlakeParams>>(Base: ActionBase) {
   return class extends Base implements ILenderActions {
     // senior tranche functions
-    submitSeniorSupplyOrder = async (user: string, supplyAmount: string) => {
+    submitSeniorSupplyOrder = async (supplyAmount: string) => {
       return this.pending(this.contract('SENIOR_OPERATOR').supplyOrder(supplyAmount, this.overrides))
```

Comment:
Again, this functions (and others from this PR) should go into Tinlake.js, right?

Reply:
What do you mean? This is Tinlake.js?

Reply:
Haha, it’s getting late - saw UI in the title and just looked at the code 🙈